include/n64/PR/os_motor.h

```diff
@@ -1,7 +1,7 @@
 
 /*---------------------------------------------------------------------*
         Copyright (C) 1998 Nintendo.
-        
+
```

Comment:
I personally think we shouldn't be touching any of the `include/n64/PR/` files with any formatting changes whatsoever, unless they revolve around code actually written by us.